src/ingest_validation_tools/table-schemas/level-2/lcms.yaml

```diff
@@ -1,13 +1,22 @@
 doc_url: TODO
 fields:
-  # Overrides of Level 1 fields:
+# Overrides of Level 1 fields:
+  -
+    name: assay_category
+    constraints:
+      enum:
+        - mass_spectrometry
+  -
+    name: assay_type
+    constraints:
+      enum:
+        - LC-MS (metabolomics)
+        - LC-MS/MS (label-free proteomics)
   -
     name: analyte_class
     constraints:
       required: False
```

Comment:
```suggestion
    name: analyte_class
    constraints:
      required: True
```

Reply:
`required: True` is the default. I'll just remove this.

Reply:
But MS can be used for any analyte? I'll leave a TODO in the code to clarify.